trust/test-builder.c

```diff
@@ -863,6 +863,71 @@ test_invalid_dates (void)
 	p11_message_loud ();
 }
 
+static void
+test_valid_false_or_time (void)
+{
+	CK_ATTRIBUTE *attrs = NULL;
+	CK_ATTRIBUTE *extra = NULL;
+	CK_RV rv;
+
+	CK_ATTRIBUTE input[] = {
+		{ CKA_NSS_SERVER_DISTRUST_AFTER, NULL, 0 },
+		{ CKA_CLASS, &certificate, sizeof (certificate) },
+		{ CKA_CERTIFICATE_TYPE, &x509, sizeof (x509) },
+		{ CKA_INVALID },
+	};
+
+	input[0].pValue = "\x00";
+	input[0].ulValueLen = 1;
+	rv = p11_builder_build (test.builder, test.index, attrs, input, &extra);
+	assert_num_eq (CKR_OK, rv);
+
+	p11_attrs_free (extra);
+	p11_attrs_free (attrs);
+	attrs = NULL;
+
+	input[0].pValue = "\061\071\060\067\060\061\060\060\060\060\060\060\132";
+	input[0].ulValueLen = 13;
+	rv = p11_builder_build (test.builder, test.index, attrs, input, &extra);
+	assert_num_eq (CKR_OK, rv);
+
```

Comment:
I suggest adding also a test for `ulValueLen == 15`

Reply:
Good idea, added.